init.fish

```diff
@@ -5,6 +5,8 @@
 # * $path          package path
 # * $dependencies  package dependencies
 
+set -q __git_plugin_initialized; and exit 0
```

Comment:
What happens when the plugin is updated?  I don't think the user would get any new changes. 

We need to add support for omf's update hook, does Fisher have similar?

Reply:
@jhillyerd Fisher doesn't have an `update` hook, it simply calls `uninstall.fish` (it doesn't support OMF `hooks/uninstall.fish` too...) and then links the `init.fish` to `conf.d`.

So what I'm having in mind is creating a `hooks/update.fish` just for OMF and it just does

```fish
source $path/uninstall.fish
source $path/init.fish  # may not be needed
```